Winterm issue on Windows 8.1 in kubectl exec

#### What happened:

After latest update (Im not doing it regularly) Im facing following error when trying to connect to GCP pods:

```
$ kubectl exec --stdin --tty ingress-nginx-controller-pxhr2 -n ingress-nginx -- bash
panic: Invalid standard handle identifier: 4294967286

goroutine 1 [running]:
k8s.io/kubernetes/vendor/github.com/Azure/go-ansiterm/winterm.GetStdFile(0xfffffff6, 0x150, 0xf)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/Azure/go-ansiterm/winterm/ansi.go:173 +0x1f6
k8s.io/kubernetes/vendor/github.com/moby/term/windows.NewAnsiReader(0xfffffff6, 0xc00000000f, 0x219ac60)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/moby/term/windows/ansi_reader.go:34 +0x36
k8s.io/kubernetes/vendor/github.com/moby/term.StdStreams(0x2198f80, 0xc0000d8000, 0x2198fa0, 0xc0000d8008, 0x0, 0x0)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/moby/term/term_windows.go:75 +0x1f8
k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/exec.(*StreamOptions).SetupTTY(0xc0000a88c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/exec/exec.go:273 +0x13d
k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/exec.(*ExecOptions).Run(0xc0000a88c0, 0x0, 0x200d550)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/exec/exec.go:331 +0x23f
k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/exec.NewCmdExec.func1(0xc0001d4dc0, 0xc000303570, 0x2, 0x7)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/exec/exec.go:95 +0x11a
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).execute(0xc0001d4dc0, 0xc000303500, 0x7, 0x7, 0xc0001d4dc0, 0xc000303500)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:854 +0x2c2
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc000392dc0, 0xc0000e8000, 0xc0000e6000, 0x9)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:958 +0x375
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:895
main.main()
        _output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/kubectl.go:49 +0x234
```

#### What you expected to happen:
#### How to reproduce it (as minimally and precisely as possible):
#### Anything else we need to know?:

I'm using GIT bash and have no issue with docker-compose EXEC, which might be bitten in similar way:
https://github.com/docker/for-win/issues/9770

#### Environment:
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-12T14:18:45Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.6-gke.1000", GitCommit:"3ae0998c5052f420a17cb96bacf860ec5d6822a3", GitTreeState:"clean", BuildDate:"2021-04-29T09:17:16Z", GoVersion:"go1.15.10b5", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration: GCP
- OS (e.g: `cat /etc/os-release`): Windows 8.1
- Kernel (e.g. `uname -a`): 
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

--- a/go.mod
+++ b/go.mod
@@ -147,7 +147,7 @@ replace (
 	cloud.google.com/go/storage => cloud.google.com/go/storage v1.6.0
 	dmitri.shuralyov.com/gpu/mtl => dmitri.shuralyov.com/gpu/mtl v0.0.0-20201218220906-28db891af037
 	github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v53.1.0+incompatible
-	github.com/Azure/go-ansiterm => github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
+	github.com/Azure/go-ansiterm => github.com/Azure/go-ansiterm v0.0.0-20210608223527-2377c96fe795
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
 	github.com/Azure/go-autorest/autorest => github.com/Azure/go-autorest/autorest v0.11.17
 	github.com/Azure/go-autorest/autorest/adal => github.com/Azure/go-autorest/autorest/adal v0.9.10
@@ -352,7 +352,7 @@ replace (
 	github.com/moby/ipvs => github.com/moby/ipvs v1.0.1
 	github.com/moby/spdystream => github.com/moby/spdystream v0.2.0
 	github.com/moby/sys/mountinfo => github.com/moby/sys/mountinfo v0.4.1
-	github.com/moby/term => github.com/moby/term v0.0.0-20201216013528-df9cb8a40635
+	github.com/moby/term => github.com/moby/term v0.0.0-20210610120745-9d4ed1856297
 	github.com/modern-go/concurrent => github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
 	github.com/modern-go/reflect2 => github.com/modern-go/reflect2 v1.0.1
 	github.com/mohae/deepcopy => github.com/mohae/deepcopy v0.0.0-20170603005431-491d3605edfb

--- a/staging/src/k8s.io/component-base/go.mod
+++ b/staging/src/k8s.io/component-base/go.mod
@@ -9,7 +9,7 @@ require (
 	github.com/go-logr/logr v0.4.0
 	github.com/google/go-cmp v0.5.4
 	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
-	github.com/moby/term v0.0.0-20201216013528-df9cb8a40635
+	github.com/moby/term v0.0.0-20210610120745-9d4ed1856297
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.26.0

--- a/staging/src/k8s.io/kubectl/go.mod
+++ b/staging/src/k8s.io/kubectl/go.mod
@@ -21,7 +21,7 @@ require (
 	github.com/liggitt/tabwriter v0.0.0-20181228230101-89fcab3d43de
 	github.com/lithammer/dedent v1.1.0
 	github.com/mitchellh/go-wordwrap v1.0.0
-	github.com/moby/term v0.0.0-20201216013528-df9cb8a40635
+	github.com/moby/term v0.0.0-20210610120745-9d4ed1856297
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
 	github.com/opencontainers/go-digest v1.0.0 // indirect

